airsonic-main/src/main/java/org/airsonic/player/Application.java

```diff
@@ -36,10 +35,17 @@
         MultipartAutoConfiguration.class, // TODO: update to use spring boot builtin multipart support
         LiquibaseAutoConfiguration.class})
 @Configuration
-@ImportResource({"classpath:/applicationContext-service.xml",
-        "classpath:/applicationContext-cache.xml",
-        "classpath:/applicationContext-sonos.xml",
-        "classpath:/servlet.xml"})
+@EnableWebMvc
```

Comment:
This annotation should likely be on the ServletConfiguration file

Reply:
Agreed.